test/unit/mdc-menu/simple.foundation.test.js

```diff
@@ -287,37 +461,30 @@ testFoundation('#close does nothing if event target has aria-disabled set to tru
     td.verify(mockAdapter.deregisterBodyClickHandler(td.matchers.anything()), {times: 0});
   });
 
-testFoundation('#close adds the animation class to start an animation', ({foundation, mockAdapter, mockRaf}) => {
+testFoundation('#close adds the animation class to start an animation', ({foundation, mockAdapter}) => {
   foundation.close();
-  mockRaf.flush();
-  mockRaf.flush();
-  td.verify(mockAdapter.addClass('mdc-simple-menu--animating'));
+  td.verify(mockAdapter.addClass(cssClasses.ANIMATING_CLOSED));
 });
 
 testFoundation('#close removes the open class from the menu', ({foundation, mockAdapter, mockRaf}) => {
-  td.when(mockAdapter.hasClass('mdc-simple-menu--open')).thenReturn(true);
-  td.when(mockAdapter.hasClass('mdc-simple-menu--open-from-bottom-left')).thenReturn(true);
-
+  td.when(mockAdapter.hasClass(cssClasses.OPEN)).thenReturn(true);
```

Comment:
Is this line even necessary, here and in the next test?

Reply:
strictly speaking it is not needed :).